startup/BMM/unreal_mono.py

```diff
@@ -0,0 +1,12 @@
+from ophyd import Device, Signal, Component
+from bluesky_unreal import UnrealSignal, UnrealPositioner
+
+class A(Device):
+    sig = Component(Signal, name='sig', kind="hinted")
+    bragg = Component(UnrealSignal, name="bragg", preset_name='mono_remote',
+                                    kind="hinted")
+    para = Component(UnrealPositioner, name="para", preset_name='mono_remote',
+                                    kind="hinted")
+
+
+a = A(name='dcm')
```

Comment:
What is this used for?  I am not seeing where this object is used.  Also, `a` and `A` are way too short/non-specific for object names in production code.

Reply:
We can delete this file.  We were just using it for testing.